package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "globals",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

Comment:
Please don't update the package version. This is done as part of the release process.